src/constants/copy.ts

```diff
@@ -0,0 +1,3 @@
+export const LISTENING_COPY = "-Can't Talk";
+
+export const DESC_MESSAGE = "Your nickname changed successfully";
```

Comment:
Please use a constant like SUCCESS_MESSAGE and ERROR_MESSAGE

Reply:
ok I will move it to response.ts